rc/base/markdown.kak

```diff
@@ -11,100 +11,32 @@ hook global BufCreate .*[.](markdown|md|mkd) %{
-    cucumber   ^```\h*cucumber\b\K   ```          '' \
-    d          ^```\h*d\b\K          ```          '' \
-    diff       ^```\h*diff\b\K       ```          '' \
-    dockerfile ^```\h*dockerfile\b\K ```          '' \
-    fish       ^```\h*fish\b\K       ```          '' \
-    gas        ^```\h*gas\b\K        ```          '' \
-    go         ^```\h*go\b\K         ```          '' \
-    haml       ^```\h*haml\b\K       ```          '' \
-    haskell    ^```\h*haskell\b\K    ```          '' \
-    html       ^```\h*html\b\K       ```          '' \
-    ini        ^```\h*ini\b\K        ```          '' \
-    java       ^```\h*java\b\K       ```          '' \
-    javascript ^```\h*javascript\b\K ```          '' \
-    json       ^```\h*json\b\K       ```          '' \
-    julia      ^```\h*julia\b\K      ```          '' \
-    kak        ^```\h*kak\b\K        ```          '' \
-    kickstart  ^```\h*kickstart\b\K  ```          '' \
-    latex      ^```\h*latex\b\K      ```          '' \
-    lisp       ^```\h*lisp\b\K       ```          '' \
-    lua        ^```\h*lua\b\K        ```          '' \
-    makefile   ^```\h*makefile\b\K   ```          '' \
-    moon       ^```\h*moon\b\K       ```          '' \
-    objc       ^```\h*objc\b\K       ```          '' \
-    perl       ^```\h*perl\b\K       ```          '' \
-    pug        ^```\h*pug\b\K        ```          '' \
-    python     ^```\h*python\b\K     ```          '' \
-    ragel      ^```\h*ragel\b\K      ```          '' \
-    ruby       ^```\h*ruby\b\K       ```          '' \
-    rust       ^```\h*rust\b\K       ```          '' \
-    sass       ^```\h*sass\b\K       ```          '' \
-    scala      ^```\h*scala\b\K      ```          '' \
-    scss       ^```\h*scss\b\K       ```          '' \
-    sh         ^```\h*sh\b\K         ```          '' \
-    swift      ^```\h*swift\b\K      ```          '' \
-    tupfile    ^```\h*tupfile\b\K    ```          '' \
-    typescript ^```\h*typescript\b\K ```          '' \
-    yaml       ^```\h*yaml\b\K       ```          '' \
-    code       ^```((!?=(c|cabal|clojure|coffee|cpp|css|cucumber|diff|dockerfile|fish|gas|go|haml|haskell|html|ini|java|javascript|json|julia|kakrc|kickstart|latex|lisp|lua|makefile|moon|objc|perl|pug|python|ragel|ruby|rust|sass|scala|scss|sh|swift|tupfile|typescript|yaml)[^\n])*)$          ```          '' \
-    code       ^``[^`]            ``           '' \
-    code       ^`[^`]             `            ''
+%sh{
+  languages="
+    c cabal clojure coffee cpp css cucumber d diff dockerfile fish gas go
+    haml haskell html ini java javascript json julia kak kickstart latex
+    lisp lua makefile markdown moon objc perl pug python ragel ruby rust
+    sass scala scss sh swift tupfile typescript yaml
+  "
+  echo 'add-highlighter shared/ regions -default content markdown \'
+  for lang in ${languages}; do
+    printf '%s    ^```\h*%s\\b   ```   "" ' "${lang}" "${lang}"
```

Comment:
Should the `\\b` here be `\K\b` ?

Reply:
No it is actually not needed anymore, the regions introduced for each language make sure that the triple-quotes are not to be highlighted by the language's highlighter.